sigopt/orchestrate/paths.py

```diff
@@ -55,17 +55,17 @@ def check_executable(command, sha256, full_check):
         raise
 
 
-KUBECTL_VERSION = 'v1.20.4'
-KUBECTL_URL_FORMAT = 'https://storage.googleapis.com/kubernetes-release/release/{}/bin/{}/amd64/kubectl'
-KUBECTL_SHA256_LINUX = '98e8aea149b00f653beeb53d4bd27edda9e73b48fed156c4a0aa1dabe4b1794c'
-KUBECTL_SHA256_MAC = '37f593731b8c9913bf2a3bfa36dacb3058dc176c7aeae2930c783822ea03a573'
+KUBECTL_VERSION = 'v1.25.2'
+KUBECTL_URL_FORMAT = 'https://dl.k8s.io/release/{}/bin/{}/amd64/kubectl'
+KUBECTL_SHA256_LINUX = '8639f2b9c33d38910d706171ce3d25be9b19fc139d0e3d4627f38ce84f9040eb'
+KUBECTL_SHA256_MAC = 'b859766d7b47267af5cc1ee01a2d0c3c137dbfc53cd5be066181beed11ec7d34'
 
 AWS_IAM_AUTHENTICATOR_URL_FORMAT = (
-  'https://github.com/kubernetes-sigs/aws-iam-authenticator/releases/download/0.4.0-alpha.1/aws-iam-'
-  'authenticator_0.4.0-alpha.1_{}_amd64'
+  'https://github.com/kubernetes-sigs/aws-iam-authenticator/releases/download/v0.5.9/aws-iam-'
+  'authenticator_0.5.9_{}_amd64'
 )
```

Comment:
Note: we should probably support ARM for `kubectl` and `aws-iam-authenticator`. This is especially relevant with a range of modern ARM platforms (ex. M1, Graviton, Raspberry PI).